src/routes/docs.js

```diff
@@ -24,14 +27,22 @@ router.get('/install', async (request, response) => {
 	response.render('docs/install', renderData);
 });
 
-router.get('/search', async (request, response) => {
+router.get([
+	'/search', // TODO - Deprecate search page
+	'/errors',
+], async (_request, response) => {
 	const renderData = {
 		errorList: JSON.stringify(errorList),
-		currentPage: 'search',
+		currentPage: 'errors',
 	};
+
 	response.render('docs/search', renderData);
 });
 
+router.get('/error', async (_request, response) => {
```

Comment:
Any particular reason for adding this? I'd be wary especially of adding a 301 as that will get cached by browsers and potentially be an issue if you ever wanted to add a page at this path in the future.

I'd recommend using a 303 if you really want to do the redirect

Reply:
No reason in particular. Just a little safety net for people who mistype the URL, since it's clear what page they'd actually want to go to (the search path is `/docs/errors`, and the support pages are at `/docs/error/{code}`. I can easily see someone just removing `{code}` and ending up at `/docs/error` but meant `/docs/errors`)

Since we have a `/docs/errors` page I don't see us making a real `/docs/error` page in the future, to avoid confusion. Though if we were to change the status code 307 may be more appropriate?

While 303 *can* be used for `GET`, it's most typically used for `POST` requests. 303 is also intended to indicate that the server does not have a representation for the requested resource, and suggests one that *may* represent it. But the server can't be sure that the resource being redirected to actually represents the one the user wants

307 on the other hand assumes to know what resource the client wants (which I would argue is the case here), and redirects them. 307 also tells the client to NOT cache the redirect and still use the original target, which sounds like a better fit than 303?

Honestly though there's not a perfect fit for a status code here if you want to reserve the path for future use

Reply:
I think 303 or 307 are both fine, the main technical difference is that 303 forces the redirected request to a GET, but 307 doesn't. In this case you're just using it on a GET endpoint anyway, sound they're functionally equivalent. I don't really have a preference between the two but think it's worth not letting the browser cache the redirect :)

Reply:
I decided to go with 307, purely for semantic reasons (I kinda sort of be a stickler there, it's a flaw I know) :+1: